lookout/style/format/tests/bugs/003_classify_vnodes_negative_col/test.py

```diff
@@ -5,14 +5,14 @@
 
 from lookout.style.format.analyzer import FormatAnalyzer
 from lookout.style.format.feature_extractor import FeatureExtractor
-from lookout.style.format.tests.test_analyzer import get_train_config
+from lookout.style.format.tests.test_analyzer import get_config
 
 
 class FeaturesTests(unittest.TestCase):
     def setUp(self):
-        config = FormatAnalyzer._load_train_config(get_train_config())
+        config = FormatAnalyzer._load_config(get_config())
         self.extractor = FeatureExtractor(language="javascript",
-                                          **config["javascript"]["feature_extractor"])
+                                          **config["train"]["javascript"]["feature_extractor"])
```

Comment:
This started to get out of control. How about using [munch](https://github.com/Infinidat/munch)? At least the string constants will go away. We could munchify everything in `_load_config`

Reply:
yes, I like it. but what about munchify everything in next PR?

Reply:
Also, I think it is still ok to have 3-level nesting because in most cases you deal only with 2 levels and this one is a test.